x11-misc/urxvt-perls/urxvt-perls-2.3-r1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Perl extensions for rxvt-unicode"
+HOMEPAGE="https://github.com/muennich/urxvt-perls"
+SRC_URI="https://github.com/muennich/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~arm x86"
```

Comment:
Since it changes list of installed files, we'd be safer by adding `~amd64 ~x86" here. You can call `ekeyword ~all urxvt-perls-2.3-r1.ebuild" to reset them.

Reply:
Ok